README.md

```diff
@@ -88,19 +94,19 @@ docker compose up -d
 
 #### Option 2: Use the latest image from Docker Hub
 ```bash
-source .env # set your environment variables 
+export $(cat .env | xargs)  # export your environment variables 
```

Comment:
interesting. were they not getting picked up as shell variables?

Reply:
The `source` command does set the variables from .env in the shell, but doesn't export them. This is probably not absolutely necessary (esp with the new variable prefixes) but more of an extra precaution to avoid conflicts with existing shell variables. 🙂